src/endpoints/sources.js

```diff
@@ -285,6 +285,18 @@ const getGroupOverview = contentTypesProvided([
 ]);
 
 
+/* PUT
+ */
+const putGroupOverviewMarkdown = contentTypesProvided([
+  ["text/markdown", receiveGroupOverview(req => req.context.source)],
+]);
+
+
+const putGroupOverview = contentTypesProvided([
+  ["text/markdown", putGroupOverviewMarkdown],
+]);
```

Comment:
Ditto about this division of handlers.

Reply:
Done: 04311615a7d85cba1f0c07bb32162e62e5b0999f

Also fixed to use `contentTypesConsumed`: 8aa733ca29f41f5eab8b2edbf8a060d966dbd937